tests/foreman/cli/test_user.py

```diff
@@ -675,6 +674,33 @@ def test_negative_create_user_4(self, email):
         self.assertNotEqual(result.return_code, 0)
         self.assertTrue(result.stderr)
 
+    @skip_if_bug_open('bugzilla', 1204686)
+    def test_bugzilla_1204686(self):
+        """@Test: Create User with Empty Email Address
+
+        @Feature: User - Negative Create
+
+        @Steps:
+        1. Create User with empty Email Address in [2]
+        using valid Username, First Name, Surname, Language, authorized by
+
+        @Assert: User is not created. Appropriate error shown.
+
+        @BZ: 1204686
+
+        """
+        options = {
+            'login': gen_string("alpha", 10),
+            'firstname': gen_string("alpha", 10),
+            'lastname': gen_string("alpha", 10),
+            'mail': '',
+            'password': gen_string("alpha", 10),
+            'auth-source-id': 1
+        }
+        result = UserObj().create(options)
+        self.assertNotEqual(result.return_code, 0)
+        self.assertTrue(result.stderr)
```

Comment:
Is `assertTrue` the best assertion to use here? How about `self.assertGreaterThan(len(result.stderr), 0)` or `self.assertIn('desired error message', result.stdout)`.